app/src/main/java/com/example/paetz/yacguide/TourbookActivity.java

```diff
@@ -161,8 +161,9 @@ public void onClick(View v) {
                     startActivityForResult(intent, 0);
                 }
             };
+
             layout.addView(WidgetUtils.createCommonRowLayout(this,
-                    rock.getName() + " - " + route.getName(),
+                    WidgetUtils.getAscendSymbol(ascend.getStyleId()) + " " + rock.getName() + " - " + route.getName(),
```

Comment:
Having done an ascend and having a project at the same time on the same rock should be possible. --> Multiple symbols would be needed here.

Reply:
OK, I will allow each symbol to show up exactly once per summit.

Reply:
Sorry, my mistake. The tourbook entry should not be affected by this.